docs/UserGuide.md

```diff
@@ -413,23 +438,39 @@ Show a list of all products in MyInsuRec.
 
 Format: `listProduct`
 
-#### 5.3.3 Delete product: `delProduct`
+Example inputs:
+1. `listProduct`
+
+Expected behavior:
+1. Displays **a list of all the products**.
 
-Delete a product from MyInsuRec.  
-This feature will remove the product from association with any client.
+Use case:
+1. Get an overview of all the products you are offering!
+2. This feature is used to get the index of a product, which is needed for most product-related commands.
+
+#### 5.3.3 Delete a product : `delProduct`
+
+Deletes a product from *MyInsuRec*.  
+This command removes this product from all the clients as well.
 
 Format: `delProduct i/INDEX`
 
-* Delete the product at the specified `INDEX`.
-* The index refers to the index number shown in the displayed product list.
-* The index **must be a positive integer** 1, 2, 3, …​
+<div markdown="span" class="alert alert-info">:information_source: **Note:**
```

Comment:
is there a rationale for using `markdown="span"` rather than `markdown="block"`? like I'm a bit confuse when to use which

Reply:
tbh no, i just copy and paste these

<img width="837" alt="image" src="https://user-images.githubusercontent.com/63795894/198701016-e1f8192b-aaec-4746-b0aa-5b2dad1dbd45.png">
<img width="840" alt="image" src="https://user-images.githubusercontent.com/63795894/198701034-7dbd696f-588a-4a1d-9262-b34515b94476.png">

here's the differences, i think its best if we use span for smaller, block for larger with the bullet points

Reply:
ahh okok